File: 프로그래머스/0/181855. 문자열 묶기/문자열 묶기.js
function solution(strArr) {
  const arr = strArr.map(x => x.length)
  const obj = arr.reduce((accu, curr) => { 
    accu[curr] = (accu[curr] || 0)+1; 
    return accu;
  }, {});
  let newArr = [];
  for (const key in obj) {
    newArr.push(obj[key])
  }
  return Math.max(...newArr)
}

File: 프로그래머스/0/181855. 문자열 묶기/문자열 묶기.java
class Solution {
    public int solution(String[] strArr) {
        int[] length = new int[31];
        
        for(String str : strArr) {
            length[str.length()]++;
        }
        
        int answer = 0;
        for(int len : length) {
            if(len > answer) {
             answer = len;
            }
        }
        return answer;
    }
}